neon_audio/tts/neon.py

```diff
@@ -134,19 +132,32 @@ def _parse_message(self, message, **kwargs):
         sentence = message.data["text"]
         responses = {}
         for request in tts_requested:
-            lang = request["language"]
-            if lang in self.cached_translations:
-                sentence = self.cached_translations[lang].get(sentence)
-            elif lang.split("-")[0] != "en":  # TODO: Internal lang DM
-                self.cached_translations[lang][sentence] = self.translator.translate(sentence, lang, "en")
-                self.cached_translations.store()
-                sentence = self.cached_translations[lang][sentence]
-                request["translated"] = True
+            lang = kwargs["lang"] = request["language"]
             wav_file, phonemes = self._get_tts(sentence, **kwargs)
```

Comment:
I think this belongs in the `else` of:
```python
            # translate if needed
            if lang.split("-")[0] != self.lang.split("-")[0]:
```

Reply:
im confused, do we not always want both ? why does it go in the else?